ui/appui-react/src/appui-react/layout/tool-settings/Docked.tsx

```diff
@@ -138,7 +138,7 @@ export function DockedToolSettings(props: DockedToolSettingsProps) {
   const PanelContainer = props.panelContainer
     ? props.panelContainer
     : DefaultPanelContainer;
-  const className = classnames("nz-toolSettings-docked", props.className);
+  const className = classnames("uifw-toolSettings-docked", props.className);
```

Comment:
@FlyersPh9  what do you think about deprecating `UiFramework.animateToolSettings`? I noticed that tool settings have some optional animations but personally I don't see any benefit from it for a few reasons: it happens only when the window is resized and this is the only place (lines 152-166 in this file) where such animations are used, so is it really needed at all?

Reply:
Unfortunately I think this needs to stay, at least for now.  This animation was added because users were not noticing the tool settings changing when switching between tools.  Even with the animation I've still heard of some users not noticing the tool settings changing.

Personally, I'm of the belief that a user may not notice this their first couple times using the application but will learn it over time making the animation no longer necessary.  Perhaps onboarding or something else could teach the user where tool settings are?

Reply:
I see, it makes sense if the animation is supposed to help users notice the tool settings. But I also agree with your point that users would easily learn where the to find the tool settings on their own. And when the users know where to find them, the animation becomes basically useless. We can leave this as it is for now.